app/hookSW.js

```diff
@@ -15,24 +15,61 @@ self.addEventListener('push', function(evt) {
   console.log('Push received ', JSON.stringify(evt.data));
   var obj = evt.data ? evt.data.json() : {};
 
-  if (obj && obj.body) {
-    showNotification(obj, evt);
+  if (obj && obj.message) {
+    notifyClient(obj)
+      .then(() => {
+        showNotification(obj, evt);
+      });
   } else {
-    console.log('Discarding notification');
+    console.log('Discarding notification: ' + JSON.stringify(obj));
```

Comment:
`console.error`, also maybe an easier message like `Notification doesn't contain a body, ignoring it`.